src/lib/modules/console/index.ts

```diff
@@ -112,9 +125,28 @@ class Console {
       this.history.push(cmd);
       this.saveHistory();
     }
-    const pluginCmds = this.plugins.getPluginsProperty("console", "console");
-    for (const pluginCmd of pluginCmds) {
-      const pluginResult = pluginCmd.call(this, cmd, {});
+    const plugins = this.plugins.getPluginsProperty("console", "console");
+    const helpDescriptions = [];
+    for (const plugin of plugins) {
+      // New API
+      if (plugin.description) {
+        helpDescriptions.push({
+          description: plugin.description,
+          matches: plugin.matches,
+          use: plugin.use,
+        });
+      }
+      if (plugin.matches) {
+        const isFunction = typeof plugin.matches === "function";
+        if ((isFunction && plugin.matches.call(this, cmd))
```

Comment:
I think there's no need to do `plugin.matches.call()`. If `plugin.matches` is a function we can just do `plugin.matches()`

Reply:
What Anthony said